gallery/experiments/experimental_abinitio_pipeline_10081.py

```diff
@@ -115,12 +119,13 @@
 # Volume Reconstruction
 # ----------------------
 #
-# Using the oriented source, attempt to reconstruct a volume.
-# Since this is a Cn symmetric molecule, as indicated by
-# ``symmetry="C4"`` above, the ``avgs`` images set will be repeated
-# for each of the 3 additional rotations during the back-projection
-# step.  This boosts the effective number of images used in the
-# reconstruction from ``n_classes`` to ``4*n_classes``.
+# Using the oriented source, attempt to reconstruct a volume.  Since
+# this is a Cn symmetric molecule, as specified by ``RelionSource(...,
+# symmetry_group="C4, ...)"``, the ``symmetry_group`` source attribute
+# will flow through the pipeline to ``avgs``. Then each image will be
```

Comment:
Turns out that the `symmetry_group` pass-through was not implemented for `ClassAvgSource`s. Added the changes in PR #1151 to merge into this PR if everything looks good.